### DISCUSSION TITLE
Postal server IP pool

### DISCUSSION THREAD

**Discussion by wakutumwa** [postalserver/postal#3237] [Help with using Postal] [answered]
Hello,
I was configuring the postal server on Redhat VM. I have reached a point IP pool needs to be configured. I have updated the entry on postal.yaml but I have some question,
1. What IP should be added to a pool, is it public IP or my vm private IP?
2. Do I have to reconfigure postal.yaml? Which entry should I add?
3. How to test if IP pool works
Thanks in advance

**Comment by willpower232** [comment, marked as answer]
> What IP should be added to a pool, is it public IP or my vm private IP?

It needs to be the IP address that the server sees in `ip a`, I don't know how it works for your server provider but if you run `ip a` you can see which IP addresses are directly available for you.

I know several server providers have different private/public IP addresses so it is likely you will need to use the private one if yours has this arrangement

> Do I have to reconfigure postal.yaml? Which entry should I add?

IP Pools are enabled in the config file and then set up using the web UI so you only have to tell the config file that you would like to use IP Pools and then the rest is in the web UI.

> How to test if IP pool works

Just send with it and see if the message can be sent without problems. I think the Send Message page in the web UI would work for this.

**Reply by wakutumwa** [reply]
Like, when I do **ip a** I see 10.10.1.10 and 10.10.1.11 but this is NATed to 163.188.123.23 which is public, so I should add 10.10.x.x to the IP pool?

**Reply by willpower232** [reply]
I think you have to specify individual IP addresses but yeah basically.

**Reply by wakutumwa** [reply]
Hello @willpower232  Help me now with this I can not get mail from Gmail, when I run that command to see my cert, it gives me 
80EBDB580E7F0080: error: 8888810B:SSL routines: ss13_get_record: wrong version number:ssl/record/ss13_record.c:354

![Screenshot 2024-11-23 203411](https://github.com/user-attachments/assets/56d41ab8-53b8-46c7-a8ce-f2c75ba104e5)

**Reply by willpower232** [reply]
Have you definitely set up an SSL certificate in the config file? and restarted Postal?

**Reply by wakutumwa** [reply]
Thank you, the issue was the ssl certificate and is fixed
